pom.xml

```diff
@@ -374,7 +374,7 @@
             <Bundle-Description>${project.description}</Bundle-Description>
             <Export-Package>org.xmlunit.*;version=${project.version};-noimport:=true</Export-Package>
             <Private-Package></Private-Package>
-            <Import-Package>*</Import-Package>
+            <Import-Package>javax.xml.bind.*;resolution=optional,*</Import-Package>
```

Comment:
This line should read
            `<Import-Package>javax.xml.bind.*;resolution:=optional,*</Import-Package>`
(with a colon).

Reply:
Local build was successful, thanks! Looks promising – will check now with our OSGi experts if the resolution problem is really solved.

Reply:
Thank you. If they say things look good I can cut a new release pretty quickly.

Reply:
Testing locally built artefacts turns out to be pretty hard in our OSGi system... A snapshot build would be easier to test. As central.sonatype... is up again, maybe you could upload another snapshot? Thanks!

Reply:
pushed a new 2.10.3-SNAPSHOT just now

Reply:
Thanks a lot for the confirmation!